recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
-{% set version = "20160523b" %}
+{% set version = "20170520" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
   fn: texlive-{{ version }}-source.tar.xz
-  url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: a8b32ca47f0a403661a09e202f4567a995beb718c18d8f81ca6d76daa1da21ed
+  url: http://mirrors.concertpass.com/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
```

Comment:
> That's a bit too far from my comfort zone

No problem. It is far from my comfort zone too :smile:

> BTW, is there a plan for how to contribute additional TexLive packages? I'll need xcolor for example.

We never settled on how to do that. A big bundle? Individual packages? Do you have any suggestion? I added this package mostly to help with `Jupyter` `nbconvert` stuff, so I never really though about the extras. (That is something I still rely on my systems package manager.)